velox/common/memory/tests/MemoryAllocatorTest.cpp

```diff
@@ -55,7 +55,16 @@ class MemoryAllocatorTest : public testing::TestWithParam<bool> {
     if (useMmap_) {
       MmapAllocator::Options options;
       options.capacity = kMaxMemoryAllocator;
+      options.smallAllocationReservePct = 4;
+      maxMallocBytes_ = options.maxMallocBytes = 3072;
       allocator_ = std::make_shared<MmapAllocator>(options);
+      auto mmapAllcator = std::dynamic_pointer_cast<MmapAllocator>(allocator_);
+      ASSERT_EQ(
+          mmapAllcator->capacity(),
+          bits::roundUp(
+              kMaxMemoryAllocator * (100 - options.smallAllocationReservePct) /
+                  100 / AllocationTraits::kPageSize,
+              64 * mmapAllcator->sizeClasses().back()));
```

Comment:
Can you add test methods to check if the fields added in MmapAllocator is initialized properly? You can have a test case that just create a MmapAllocator instance and tests its states? Thanks!

Reply:
I'll add an additional independent test for it